docs/faq/code-analysis/which-metrics-does-codacy-calculate.md

```diff
@@ -0,0 +1,124 @@
+---
+description: Codacy scans your code for issues and calculates the metrics code complexity, duplication, and coverage. Besides this, Codacy also calculates a grade for your repository and files based on all calculated code quality metrics.
+---
+
+# Which metrics does Codacy calculate?
+
+Codacy performs static code analysis and calculates code duplication, code complexity, and code coverage metrics for [most supported programming languages](../../getting-started/supported-languages-and-tools.md).
+
+The following sections describe how Codacy calculates each supported metric and where you can see each metric on the Codacy UI:
+
+-   [Grade](#grade)
+-   [Issues](#issues)
+-   [Complexity](#complexity)
+-   [Duplication](#duplication)
+-   [Code coverage](#code-coverage)
+
+!!! note
+    Depending on certain characteristics of your repository, such as the number of source code files and their number, Codacy may [apply limits to the code analysis](does-codacy-place-limits-on-the-code-analysis.md) that impact the calculation of the supported metrics.
+
+## Grade
+
+Codacy assigns an overall grade to your repository branches and to individual files to help you understand the code quality of your repository. Grades depend on a score that takes into account a weighted average of all supported code quality metrics, and range from **A** (highest grade) to **F** (lowest grade):
+
+|Grade                          |Score     |
+|-------------------------------|----------|
+|![Grade A](images/grade_a.png) |91 to 100 |
+|![Grade B](images/grade_b.png) |71 to 90  |
+|![Grade C](images/grade_c.png) |51 to 70  |
+|![Grade D](images/grade_d.png) |31 to 50  |
+|![Grade E](images/grade_e.png) |11 to 30  |
+|![Grade F](images/grade_f.png) |0 to 10   |
+
+Codacy displays grades on the following pages:
```

Comment:
not sure if relevant, but some emails also include a grade (commit and pull request?)

Reply:
Indeed, and besides, this is also valid for the other metrics. I'll update the page to also mention the email notifications for each metric.